script/AggregationEditor.js

```diff
@@ -140,7 +142,7 @@ const AggregationEditor = function (idx, table) {
         DOKU_BASE + 'lib/exe/ajax.php',
         {
             call: 'plugin_struct_aggregationeditor_new',
-            schema: schema
+            data: {schema: schema, searchconf: searchconf}
```

Comment:
Isn't the schema part of the SearchConfig? Wouldn't it be enough to pass the latter param?

Reply:
SearchConf contains an array of schemas and I'd rather not refactor the code at this point.

Reply:
Hmm. We only support a single schema anyway, right? Can't you simply access searchconf->schemas[0]? Maybe I misunderstand.